jujugui/static/gui/src/app/app.js

```diff
@@ -2113,7 +2112,6 @@ YUI.add('juju-gui', function(Y) {
    */
     _ensureControllerConnection: function(state, next) {
       if (
-        state.root !== 'logout' &&
```

Comment:
It's actually actively bad--the logout mechanisms need an active websocket. So we need to have a connection to logout. Which is peculiar, but not actually bad.